sources/newRif/RefactorRawFiles.py

```diff
@@ -30,7 +34,7 @@ def parse_paragraphs(page: str) -> list:
 
 
 def prepare_output(segment: str, output_list: list, parsing_state: ParseState) -> None:
-    if PARSING_STATE.get_ref('paragraph') == 1:
+    if parsing_state.get_ref('paragraph') == 0:
         TEXT_OUTPUT.append('@20' + encode_hebrew_daf(section_to_daf(PARSING_STATE.get_ref('page')+1)))
```

Comment:
We need to remove ALL the references to global variables, not just the one in the line where I make a comment.